app/models/reaction.rb

```diff
@@ -93,7 +93,7 @@ def bust_reactable_cache
   end
 
   def async_bust
```

Comment:
This needs to be moved to an `after_commit` callback to ensure that when a reaction is created we can find it and bust its Article cache accordingly.

Reply:
Sure @mstruve